examples/applications/plot_cyclical_feature_engineering.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
From my understanding, utf-8 is the default encoding for Python 3. Given that, I would prefer to remove it everywhere. @DimitriPapadopoulos What do you think?

Reply:
Done.